packages/@aws-cdk/aws-lambda-event-sources/lib/stream.ts

```diff
@@ -21,6 +21,47 @@ export interface StreamEventSourceProps {
    */
   readonly batchSize?: number;
 
+  /**
+   * If the function returns an error, split the batch in two and retry.
+   *
+   * @default false
+   */
+  readonly bisectBatchOnFunctionError?: boolean;
+
+  /**
+   * An Amazon SQS queue or Amazon SNS topic destination for discarded records.
+   *
+   * @default discarded records are ignored
+   */
+  readonly destinationConfig?: lambda.CfnEventSourceMapping.DestinationConfigProperty | IResolvable | undefined;
+
+  /**
+   * The maximum age of a record that Lambda sends to a function for processing.
+   * Valid Range:
+   * * Minimum value of 60
+   * * Maximum value of 604800
+   *
+   * @default 604800
+   */
+  readonly maximumRecordAgeInSeconds?: number;
+
+  /**
+   * Maximum number of retry attempts
+   *
+   * @default todoNumber
```

Comment:
Maybe this should be 10000, like how maximumRecordAgeInSeconds is the maximum.

Reply:
Woops - good catch.  Never went to backfill this number